admin_guide/http_proxies.adoc

```diff
@@ -0,0 +1,166 @@
+= Working with HTTP Proxies
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: This should be a clear and concise description of the topic.
+:keywords: comma, separated, keywords, list
+
+toc::[]
+
+== Overview
+Production environments can deny direct access to the web and instead have an
+HTTP or HTTPS proxy available. Configuring OpenShift builds and deployments to
+use these proxies can be as simple as setting standard environment variables in
+configuration or JSON files.
+
+[[configuring-hosts-for-proxies]]
+
+== Configuring Hosts for Proxies
+
+. Add the `*NO_PROXY*`, `*HTTP_PROXY*`, and `*HTTPS_PROXY*` environment variables
+to each host's *_/etc/sysconfig/openshift-master_* or
+*_/etc/sysconfig/openshift-node_* file depending on the type of host:
+
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+. Restart the master or node as appropriate:
++
+----
+# systemctl restart openshift-master
+# systemctl restart openshift-node
+----
+
+[[proxying-docker-pull]]
+
+== Proxying Docker Pull
+
+Docker pulls content from the registry that hosts your built images. If you have
+an existing internal registry, use the proxy's IP with the NO_PROXY setting.
+This blacklists that registry's address, leaving the external http proxy as the
+only option.
+
+. Edit the *_/etc/sysconfig/docker_* file and add the variables in shell format:
++
+====
+----
+HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
+HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
+NO_PROXY=master.hostname.example.com
+----
+====
+
+. Restart the master, node, and Docker services:
++
+----
+# systemctl restart openshift-master
+# systemctl restart openshift-node
+# systemctl restart docker
+----
+
+. If you had already imported ImageStreams before creating the proxy
+configuration, delete, then re-run the process:
++
```

Comment:
This (ImageStreams) does not belong here - I believe it's a leftover since the latest change of adding a section for default templates below. In other words: I think this should be just removed.